style.css

```diff
@@ -0,0 +1,268 @@
+    margin: 0;
+    font-size:2.4rem;
+    padding: 40px 0 0 0;
+}
+header h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size:1.8rem;
+    font-weight:600;
+    margin: 0;
+    padding:20px 0 10px 0;
+}
+header p {
+    font-family: 'Mukta', sans-serif;
+    font-size:0.8rem;
+    width:50%;
+    margin:0;
+    padding:0 0 10px 0;
+}
+#badges {
+    display: flex;
+    -ms-display:flex;
+    padding: 8px 0 20px 0;
+    font-size:2.6rem;
+}
+#showcase {
+    margin: 0;
+    padding: 0;
+    background-color: #bbbbbb;
+    width:100%;
+}
+#showcasewrapper {
+    width:65%;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    margin: 0 auto;
+}
+#showcasetext {
+    width:40%
+}
+#showcasetext h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size:1.8rem;
+    font-weight:600;
+    margin: 0;
+    padding:40px 0 10px 0;
+}
+#showcasetext h3 {
+    font-family: 'Poppins', sans-serif;
+    font-size:0.75rem;
+    width:90%;
+    margin:0;
+    padding:4px 0 4px 0;
+}
+#showcasetext p {
+    font-family: 'Mukta', sans-serif;
+    font-size:0.9rem;
+    width:90%;
+    margin:0;
+    padding:4px 0 40px 0;
+}
+#showcaseviewer {
+    width:60%;
+    height:266px;
+    border: 8px solid #111111;
+    border-radius: 6px;
+    margin:40px 0 40px 0;
+    background-image: url(projectexample.jpg);
+    background-size: 100% auto;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+.showcasenav {
+    margin: 20% 0 25% 0;
+    background-color:rgba(102, 102, 102, 0.66);
+    color: #ffffff;
+    width: 40px;
+    height:30px;
+    border:0;
+}
+#showcaseleft {
+    float:left;
+}
+#showcaseright {
+    float:right;
+}
+#showcasebottom {
+    margin-top:224px;
+    background-color:rgba(51, 51, 51, 0.66);
+    text-align:center;
+}
+#showcaseview {
+    background-color:transparent;
+    color: #ffffff;
+    border:0;
+    font-size:1.4rem;
+}
+#contact {
+    margin: 0;
+    padding: 0;
+    background-color: #222222;
+    width: 100%;
+    color:#ffffff;
+}
+#contactwrapper {
+    width:65%;
+    margin:0 auto;
+}
+ul {
+    list-style-type: none;
+    margin:0 auto;
+    padding:10px 0 10px 0;
+}
+ul>li{
+    padding: 2px 0 2px 0;
+    line-height:0.8rem;
+    font-size:0.8rem;
+    font-family: 'Mukta', sans-serif;
+}
+@media screen and (max-width: 992px) {
+    #headertext p {
+        width: 70%;
+    }
+    #showcasetext {
+        width: 100%;
+    }
+    #showcasetext p {
+        padding: 4px 0 20px 0;
+    }
+    #showcaseviewer {
+        width: 100%;
+        margin:20px 0 40px 0;
+    }
+    #showcasewrapper {
+        -webkit-box-orient: vertical;
+        -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+        flex-direction: column;
+    }
+}
+@media screen and (max-width: 674px) {
+    #navwrapper {
+        width:78.5%;
```

Comment:
you should comma seperate all the selectors with the same css here:
```
#navwrapper,
#headerwrapper,
#showcasewrapper {
```

Reply:
resolved in e4c70e8